models/topics.js

```diff
@@ -128,5 +129,10 @@ exports.selectUsers = () => {
     })
 }
 
-
-
+exports.selectApi  = async() => {
+    return await readFile('./endpoints.json', 'utf8')
+    .then((endpoints) => {
+        const parsedEndpoints = JSON.parse(endpoints)
+        return JSON.stringify(parsedEndpoints)
+    })
+}
```

Comment:
For this, we should actually not need the models at all!

Reply:
removed this function from models!